tests/Agent/IntegrationTests/UnboundedIntegrationTests/Oracle/EnterpriseLibraryOracleTests.cs

```diff
@@ -43,15 +43,19 @@ public EnterpriseLibraryOracleTests(RemoteServiceFixtures.OracleBasicMvcFixture
                 },
                 exerciseApplication: () =>
                 {
+                    _fixture.CreateTable();
+
                     _fixture.GetEnterpriseLibraryOracle();
                     _fixture.AgentLog.WaitForLogLine(AgentLogBase.AgentConnectedLogLineRegex, TimeSpan.FromMinutes(1));
                     _fixture.AgentLog.WaitForLogLine(AgentLogBase.SqlTraceDataLogLineRegex, TimeSpan.FromMinutes(1));
+
+                    _fixture.DropTable();
```

Comment:
If an exception occurs prior to this line, the table used by the test will not be cleaned up.